### ISSUE TITLE
Warnings rush after upgrade to version 0.4.0

### ISSUE THREAD

**Issue by take-five** [mattbrictson/airbrussh#10]
Hello, hope you're well.

I used 0.3.0 version of your gem in my projects and everything seemed to be fine, but after upgrade I began to get tons of warnings:
```
/Users/amikhailov/.rvm/gems/ruby-2.1.5/gems/airbrussh-0.4.0/lib/airbrussh/formatter.rb:230: warning: Capistrano::Configuration::Server::Properties#respond_to?(:marshal_dump) is old fashion which takes only one parameter
/Users/amikhailov/.rvm/gems/ruby-2.1.5/gems/capistrano-3.1.0/lib/capistrano/configuration/server.rb:81: warning: respond_to? is defined here
/Users/amikhailov/.rvm/gems/ruby-2.1.5/gems/airbrussh-0.4.0/lib/airbrussh/formatter.rb:230: warning: Capistrano::Configuration::Server::Properties#respond_to?(:_dump) is old fashion which takes only one parameter
/Users/amikhailov/.rvm/gems/ruby-2.1.5/gems/capistrano-3.1.0/lib/capistrano/configuration/server.rb:81: warning: respond_to? is defined here
```

After digging into a code I've found out that these warnings are generated by this method in `Airbrussh::Formatter`:
```
    def deep_copy(obj)
      Marshal.load(Marshal.dump(obj))
    end
```

After that I've looked into Capistrano code:
```
        def respond_to?(method)
          @properties.has_key?(method)
        end
```

Obviously, these warnings come from old Capistrano which is not suited well to new ruby. I'm not certain if they fixed this in newer versions however. Nevertheless I cannot upgrade to newer Capistrano at this moment (capistrano comes as dependency of internal gem).

I know it is not your fault but maybe there is another way to implement `deep_copy` stuff having in mind some legacy code (like old capistrano, for instance)?

**Comment by robd** [comment]
Hi @take-five 

Thanks for reporting this issue. We are in the process of fixing this. The need to deep copy is now [removed in SSHKit master](https://github.com/robd/sshkit/commit/e93f819cbd1201179a4e27e5e8dfec50c37c6280) and hopefully @mattbrictson will get a chance to remove the `deep_copy` Airbrussh at some point soon.

**Comment by mattbrictson** [comment]
Thanks for the bug report! I missed this one since I had not tested against older Capistrano.

In any case, as @robd mentioned, `deep_copy` is no longer needed now that SSHKit master was updated earlier today with a fix for the underlying issue.

I've removed `deep_copy` and will release airbrussh 0.4.1 in the next few minutes.

**Comment by mattbrictson** [comment]
Fixed in 66271b1.

**Comment by take-five** [comment]
Thanks for quick fixing this, works like a charm now :)
